runtime/ztests/expr/in-subquery.yaml

```
@@ -0,0 +1,17 @@
+script: |
+  super -s -c 'values "foo" in (values "foo", "bar", "baz")'
+  echo // ===
+  super -s -c 'values "foo" not in (values "fool")'
+  echo // ===
+  super -s -c 'values "foo" in (values "fool")'
+
+vector: true
+
+outputs:
+  - name: stdout
+    data: |
+      true
+      // ===
+      true
+      // ===
+      false
```

Nit: Make this a spq-style test.

```suggestion
spq: |
  values
    "foo" in (values "foo", "bar", "baz"),
    "===",
    "foo" not in (values "fool"),
    "===",
    "foo" in (values "fool")

vector: true

output: |
  true
  "==="
  true
  "==="
  false
```